src/jit/codegenarm64.cpp

```diff
@@ -1135,6 +1135,10 @@ void                CodeGen::genCaptureFuncletPrologEpilogInfo()
 
     unsigned saveRegsCount = genCountBits(rsMaskSaveRegs);
     unsigned saveRegsPlusPSPSize = saveRegsCount * REGSIZE_BYTES + /* PSPSym */ REGSIZE_BYTES;
+    if (compiler->info.compIsVarArgs)
+    {
+        saveRegsPlusPSPSize += MAX_REG_ARG * REGSIZE_BYTES;
```

Comment:
You could add a comment that explains that for varargs we always save all of the integer register arguments, so that they are contiguous with the incoming stack arguments. That's why MAX_REG_ARGS is used here.